config/auto_battle_operation/席德-强化特殊技.sample.yml

```
@@ -3,7 +3,10 @@ operations:
     state: "自定义-动作不打断"
     seconds: 3.5
-  - op_name: "按键-特殊攻击"
+  - op_name: "设置状态"
+    state: "自定义-无视闪光"
+    seconds: 3.5
   - op_name: "按键-特殊攻击-按下"
   - op_name: "等待秒数"
-    seconds: 3.5
\ No newline at end of file
+    seconds: 3.5
+  - op_name: "按键-特殊攻击-松开"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)